src/main/java/seedu/staffsnap/model/applicant/NameContainsKeywordsPredicate.java

```diff
@@ -19,7 +18,7 @@ public NameContainsKeywordsPredicate(List<String> keywords) {
     @Override
     public boolean test(Applicant applicant) {
         return keywords.stream()
-                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(applicant.getName().fullName, keyword));
+                .anyMatch(keyword -> applicant.getName().fullName.toLowerCase().contains(keyword.toLowerCase()));
```

Comment:
Perhaps can consider abstracting away this check within the StringUtil class which was created for this purpose.

One possible way this can be done is through a slight modification of the existing containsWordIgnoreCase() method into a containsStringIgnoreCase() method. Your call! 😀